core/src/main/java/org/incendo/interfaces/core/Interface.java

```diff
@@ -82,6 +82,7 @@
 
         <S> Builder<T, U, V> addTransform(
```

Comment:
we should probably add a good description here explaining what the property and priority parameters do